src/sagemaker/jumpstart/curated_hub/types.py

```diff
@@ -12,12 +12,39 @@
 # language governing permissions and limitations under the License.
 """This module stores types related to SageMaker JumpStart CuratedHub."""
 from __future__ import absolute_import
-from typing import Dict, Any, Optional
+from typing import Dict, Any, Optional, List
 from enum import Enum
 from dataclasses import dataclass
 from datetime import datetime
 
-from sagemaker.jumpstart.types import JumpStartDataHolderType, JumpStartModelSpecs
+from sagemaker.jumpstart.types import JumpStartDataHolderType, JumpStartModelSpecs, HubContentType
+
+class CuratedHubTagName(str, Enum):
+    """Enum class for Curated Hub tag names."""
+    DEPRECATED_VERSIONS = "deprecated_versions"
+    TRAINING_VULNERABLE_VERSIONS = "training_vulnerable_versions"
+    INFERENCE_VULNERABLE_VERSIONS = "inference_vulnerable_versions"
+
+@dataclass
+class HubContentSummary:
+  """Dataclass to store HubContentSummary from List APIs."""
+  hub_content_arn: str
+  hub_content_name: str
+  hub_content_version: str
+  hub_content_type: HubContentType
+  document_schema_version: str
+  hub_content_status: str
+  creation_time: str
+  hub_content_display_name: str = None
+  hub_content_description: str = None
+  hub_content_search_keywords: List[str] = None
+  
+
+@dataclass
+class CuratedHubTag:
+  """Dataclass to store Curated Hub-specific tags."""
+  key: CuratedHubTagName
+  value: str
```

Comment:
Okay, I'll disagree and commit :). But if you're going to keep this, why not make the `key/value` be capitalized so it's compatible with `sagemaker:AddTags`?